src/pages/usage.md

```diff
@@ -0,0 +1,101 @@
+| :-------- | :----------- | -------------: |
+| 3.8-beta1 | 2024-02-21    |         2,829 |
+| 3.7.9     | 2024-02-10    |        21,758 |
+| 3.7.7     | 2023-11-29    |        34,715 |
+| 3.7.6     | 2023-10-09    |        32,908 |
+| 3.7.5     | 2023-09-11    |        17,934 |
+| 3.7.4     | 2023-07-17    |        25,012 |
+| 3.7.3     | 2023-06-26    |         9,940 |
+| 3.7.2     | 2023-04-05    |        41,621 |
+| 3.7.1     | 2023-03-10    |        15,576 |
+| 3.7.0     | 2023-02-15    |        14,573 |
+| 4.0-alpha1| 2021-12-30    |           286 |
+| 3.5.2     | 2022-01-26    |        73,861 |
+| 3.5.1     | 2021-12-19    |        11,676 |
+| 3.5.0     | 2021-11-07    |        18,143 |
+| 3.4.1     | 2020-09-24    |      149,121 |
+| 3.4       | 2020-09-06    |         9,390 |
+
+## Academic Citations 
+OpenRefine is used by many academic in their research and cited in their publication. OpenRefine is also available on Zenodo with the [DOI-10.5281](https://zenodo.org/records/10689569) if you intent to cite it. The table below track the number of citation per year based by searching the following terms on Google Scholar:
+* [Google Refine](https://scholar.google.ca/scholar?hl=en&as_sdt=0%2C5&as_ylo=2023&as_yhi=2023&q=%22Google+Refine%22+-openrefine&btnG=)
+* [Open Refine](https://scholar.google.ca/scholar?hl=en&as_sdt=0%2C5&as_ylo=2023&as_yhi=2023&q=%22Open+Refine%22+-openrefine&btnG=)
+* [OpenRefine](https://scholar.google.ca/scholar?hl=en&as_sdt=0%2C5&as_ylo=2023&as_yhi=2023&q=%22OpenRefine%22&btnG=)
+
+
+```mermaid 
+%%{init: {'theme':'forest'}}%%
+xychart-beta
+    title "Number of academic citations of OpenRefine per years"
+    x-axis [2010, 2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023]
+    y-axis "Count of citation" 0 --> 900
+    bar [4, 67, 162, 251, 356, 424, 477, 502, 584, 571, 586, 717, 783, 811]
+```
+
+## Forum Statistic 
+
+On November 2022, we moved from our Google Groups to discourse to host our forum. 
+
+As of March 20, 2024, over the last 12 months
+* 244 new users signed up on our forum for a total of 470 users. 
+* 324 topics were created for a total of 1,700 messages.
+
+## Contribution Statistics
+
+The statistics below only track activity on our primary GitHub repository, which includes code, design, and translation contributions. Our documentation is available in a separate repository.
+
+As of March 20th, 2024, over the last 12 months we have: 
+* 43 active github contributors (including dependabot and weblate);
+* [226 issues created](https://github.com/OpenRefine/OpenRefine/issues?q=is%3Aissue+created%3A%3E2023-03-13+);
+* [278 PR created](https://github.com/OpenRefine/OpenRefine/pulls?q=is%3Apr+created%3A%3E2023-03-13) (excluding PR created by [dependabot](https://github.com/OpenRefine/OpenRefine/pulls?q=is%3Apr+created%3A%3E2023-03-13+dependabot)) and [263 merged](https://github.com/OpenRefine/OpenRefine/pulls?q=is%3Amerged+created%3A%3E2023-03-13+);
```

Comment:
```suggestion
* [223 PRs merged](https://github.com/OpenRefine/OpenRefine/pulls?page=3&q=is%3Amerged+created%3A2023-03-20..2024-03-20+-author%3Aapp%2Fdependabot) (excluding those created by dependabot) 
```
Merged (excluding bots) is what counts here. Also update the dates in the query to match what the text above says.